Core/Src/main.c

```diff
@@ -65,41 +67,86 @@ void SystemClock_Config(void);
 int main(void)
 {
   /* USER CODE BEGIN 1 */
-
-  /* USER CODE END 1 */
-
-  /* MCU Configuration--------------------------------------------------------*/
-
-  /* Reset of all peripherals, Initializes the Flash interface and the Systick. */
-  HAL_Init();
-
-  /* USER CODE BEGIN Init */
-
-  /* USER CODE END Init */
-
-  /* Configure the system clock */
-  SystemClock_Config();
-
-  /* USER CODE BEGIN SysInit */
-
-  /* USER CODE END SysInit */
-
-  /* Initialize all configured peripherals */
-  MX_GPIO_Init();
-  MX_USART2_UART_Init();
-  /* USER CODE BEGIN 2 */
-
-  /* USER CODE END 2 */
-
-  /* Infinite loop */
-  /* USER CODE BEGIN WHILE */
-  while (1)
-  {
-    /* USER CODE END WHILE */
-
-    /* USER CODE BEGIN 3 */
-  }
-  /* USER CODE END 3 */
+    //Transmit Bytes
+    //First byte: 00000001 -> 1
+    //Second byte: 1000000 -> 128
+    //Third byte: doesn't matter -> 0
+    uint8_t tx_data[3] = {1, 128, 0};
```

Comment:
Would be best to represent this value in hex. This is mainly convention and for readability because we are literally representing bits to send.

Reply:
Also, if tx_data does not change, it should be const.